autoload/nvimdev.vim

```diff
@@ -49,11 +56,15 @@ function! nvimdev#init(path) abort
       autocmd BufWritePost *.c,*.h,*.lua call s:build_db()
     endif
     if get(g:, 'nvimdev_build_readonly', 1)
-      execute 'autocmd BufRead ' . s:path . '/build/* setlocal readonly nomodifiable'
-      execute 'autocmd BufRead ' . s:path . '/.deps/* setlocal readonly nomodifiable'
+      execute 'autocmd BufEnter '.fnameescape(s:path).'/{build,.deps}/* call s:set_readonly()'
```

Comment:
`b: _nvimdev_set_readonly` could be avoided by using `++once` (untested):

    execute 'autocmd BufEnter '.fnameescape(s:path).'/{build,.deps}/* au CursorMoved <buffer> ++once setlocal readonly nomodifiable

Reply:
Cool, thanks!
Does not work with stable nvim however.

Reply:
Changed to use it with nvim-0.4.0 only.
Good enough for me.